src/main/java/com/devcycle/sdk/server/common/model/User.java

```diff
@@ -32,77 +37,68 @@ public class User {
+  @JsonProperty("email")
   private String email;
 
   @Schema(description = "User's name used to identify the user on the dashboard / target audiences")
+  @JsonProperty("name")
   private String name;
 
   @Schema(description = "User's language in ISO 639-1 format")
+  @JsonProperty("language")
   private String language;
 
   @Schema(description = "User's country in ISO 3166 alpha-2 format")
+  @JsonProperty("country")
   private String country;
 
   @Schema(description = "App Version of the running application")
+  @JsonProperty("appVersion")
   private String appVersion;
 
   @Schema(description = "App Build number of the running application")
+  @JsonProperty("appBuild")
   private String appBuild;
 
   @Schema(description = "User's custom data to target the user with, data will be logged to DevCycle for use in dashboard.")
+  @JsonProperty("customData")
   private Object customData;
 
   @Schema(description = "User's custom data to target the user with, data will not be logged to DevCycle only used for feature bucketing.")
+  @JsonProperty("privateCustomData")
   private Object privateCustomData;
 
   @Schema(description = "Date the user was created, Unix epoch timestamp format")
+  @JsonProperty("createdDate")
+  @JsonDeserialize(using = LongTimestampDeserializer.class)
   private Long createdDate;
 
   @Schema(description = "Date the user was last seen, Unix epoch timestamp format")
+  @JsonProperty("lastSeenDate")
+  @JsonDeserialize(using = LongTimestampDeserializer.class)
   private Long lastSeenDate;
 
   @Schema(description = "Platform the SDK is running on")
-  private String platform;
+  @Builder.Default
+  @JsonProperty("platform")
+  private String platform = "Java";
 
   @Schema(description = "Version of the platform the SDK is running on")
-  private String platformVersion;
-
-  @Schema(description = "User's device model")
-  private String deviceModel;
+  @Builder.Default
+  @JsonProperty("platformVersion")
+  private String platformVersion = System.getProperty("java.version");
 
   @Schema(description = "DevCycle SDK type")
-  private SdkTypeEnum sdkType;
+  @Builder.Default
+  @JsonProperty("sdkType")
+  private PlatformData.SdkTypeEnum sdkType = PlatformData.SdkTypeEnum.SERVER;
 
   @Schema(description = "DevCycle SDK Version")
-  private String sdkVersion;
-
-  public enum SdkTypeEnum {
-    API("api"),
-    SERVER("server");
-
-    private final String value;
-
-    SdkTypeEnum(String value) {
-      this.value = value;
-    }
-
-    @JsonValue
-    public String getValue() {
-      return value;
-    }
-
-    @Override
-    public String toString() {
-      return String.valueOf(value);
-    }
-
-    public static SdkTypeEnum fromValue(String text) {
-      for (SdkTypeEnum b : SdkTypeEnum.values()) {
-        if (String.valueOf(b.value).equals(text)) {
-          return b;
-        }
-      }
-      return null;
-    }
+  @Builder.Default
+  @JsonProperty("sdkVersion")
+  private String sdkVersion = "1.1.0";
```

Comment:
Anyway to pull this from the Gradle or something so we don't have to manually update this all the time?

Reply:
Getting the sdkVersion from the build.gradle file now

Reply:
looks like its still manually defined here? any way to have this populate the version during build time? or dynamically at run-time?

Reply:
for example: https://stackoverflow.com/questions/33020069/how-to-get-version-attribute-from-a-gradle-build-to-be-included-in-runtime-swing

Reply:
I've followed this: https://stackoverflow.com/questions/37642837/gradle-make-build-version-available-to-java
So we're getting the sdkVersion this way: `User.class.getPackage().getImplementationVersion()`